### DISCUSSION TITLE
Nuxt 3.2.0 support

### DISCUSSION THREAD

**Discussion by pablosdos** [Akryum/v-tooltip#931] [General]
In Nuxt3 (3.2.0) buildModules are deprecated. How can I set it up?

**Comment by arashsheyda** [comment]
as you can see in the docs, it says `Nuxt and Nuxt Modules are now build-time-only.` so yo can move all your `buildModules` to `modules`
https://nuxt.com/docs/migration/configuration#modules

**Comment by pablosdos** [comment]
There is a solution which works for me, not adding it to Nuxt config at all.

https://github.com/Akryum/floating-vue/issues/820#issuecomment-1031077939

**Reply by arashsheyda** [reply]
Yes, you can add it as a plugin `plugins/floating-vue.ts`

```
import FloatingVue from 'floating-vue' 
 import { defineNuxtPlugin } from '#app' 
  
 export default defineNuxtPlugin((nuxtApp) => { 
   nuxtApp.vueApp.use(FloatingVue) 
 })

```
